repose-aggregator/components/filters/uri-normalization/src/test/java/org/openrepose/filters/urinormalization/UriNormalizationHandlerFactoryTest.java

```diff
@@ -53,4 +57,23 @@ public void shouldCreateNewInstanceOfContentNormalizationHandler() throws Except
         UriNormalizationHandler handler = instance.buildHandler();
         assertNotNull("Instance of Content Normalization Handler should not be null", handler);
     }
+
+    @Test
+    public void shouldInitializeMediaTypeListenerWhenNoMediaVariantsProvided() throws Exception {
+        HttpServletRequest mockRequest = mock(HttpServletRequest.class);
+        ReadableHttpServletResponse mockResponse = mock(ReadableHttpServletResponse.class);
+
+        UriNormalizationConfig config = new UriNormalizationConfig();
+        instance.configurationUpdated(config);
+
+        UriNormalizationHandler handler = instance.buildHandler();
+
+        try {
```

Comment:
Tests automatically fail if they throw an exception that wasn't explicitly mentioned in the test annotation